doc/source/handling-dependencies.rst

```diff
@@ -319,6 +319,13 @@ The ``runtime_env`` is a Python dictionary including one or more of the followin
 
   - Example: ``{"OMP_NUM_THREADS": "32", "TF_WARNINGS": "none"}``
 
+- ``container`` (dict): Require a given (Docker) image, and the worker process will run in a container with this image.
+  The `worker_path` is the default_worker.py path. The `run_options` list spec is here: https://docs.docker.com/engine/reference/run/.
+
+  - Example: ``{"image": "anyscale/ray-ml:nightly-py38-cpu", "worker_path": "/root/python/ray/workers/default_worker.py", "run_options": ["--cap-drop SYS_ADMIN","--log-level=debug"]}``
```

Comment:
`worker_path` is required only if ray installation directory in the container is different from raylet host.

Reply:
Already added.